packages/console/src/pages/Settings/index.tsx

```diff
@@ -59,14 +41,16 @@ const Settings = () => {
       <TabNav>
         <TabNavItem href="/settings">{t('settings.tabs.general')}</TabNavItem>
       </TabNav>
-      {!data && !error && <div>loading</div>}
-      {!data && error && <div>{`error occurred: ${error.body?.message ?? error.message}`}</div>}
-      {data && (
+      {isLoading && <div>loading</div>}
+      {!isLoading && error && (
+        <div>{`error occurred: ${error.body?.message ?? error.message}`}</div>
+      )}
+      {isLoaded && (
```

Comment:
nit: Is `isLoaded` necessary?

Reply:
Sorry didn't follow this. Why is it unnecessary?

Reply:
If it's not loading, nor having error, it should be `loaded` already. I think we can simply return in this case

Reply:
these three conditions are in parallel - didn't see return here

Reply:
updated error display logic though, should be more clear now